### PR TITLE
Fine tune doc for eventing

### PR DESCRIPTION
**Pull request by sayanh**
**Description**

Changes proposed in this pull request:

- Fine tune doc as mentioned in https://kyma-community.slack.com/archives/CD1C9GZMK/p1558275203026400

### CODE REVIEW

File: docs/event-bus/02-01-event-bus.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ For example, whenever the `order-created` Event comes in, the Event Bus stores i
 
 
 1. A user creates a lambda or a service that an Event coming from an external solution triggers. 
-    >**NOTE**: For a service, the user must create a Kyma Subscription resource manually. For a lambda, it is created automatically.
+    >**NOTE**: For a service, the user must create a Kyma Subscription resource manually. If the user is creating a lambda in Kyma Console UI then it is created automatically else the user needs to create the Kyma Subscription manually.
```
Comment (bszwarc) [inline docs/event-bus/02-01-event-bus.md]:
```suggestion
    >**NOTE**: When creating a service, the user must create a Kyma Subscription resource manually. If the user uses Kyma Console UI to create a lambda, the Subscription resource is created automatically. In any other case, Kyma Subscription must be created manually.
```

Comment (sayanh) [reply docs/event-bus/02-01-event-bus.md]: Changed

**Pull request merged**